flake.nix

```diff
@@ -24,6 +24,10 @@
           overlays = [
             inputs.nix-ros-overlay.overlays.default
             inputs.nixgl.overlay
+            (final: prev: {
+              # Override protobuf version to ensure compatibility
```

Comment:
ok, but compatibility with what ?
We'll need to know when we can remove that

Reply:
I guess this is for Ignition Gazebo, but I might be wrong